KNN/Network_Base.cpp

```diff
@@ -1,20 +1,21 @@
 #include "Network_Base.h"
 
-
-
 Network_Base::Network_Base(size_t dz, size_t iz, size_t oz, unique_ptr<float[]> &data_ref) :
 	data_size(dz), input_size(iz), output_size(oz)
 {
 	data.reset(data_ref.get()); 
 
-	if (data_size - output_size == input_size) {  //if data size and input layer are the same size each neuron has just one input
+	if (data_size - output_size == input_size)
+	{  //if data size and input layer are the same size each neuron has just one input
```

Comment:
I see why you did these changes! I still think these are personal preferences and would not like to merge this. I was trying to use the [1TBS  ](https://en.wikipedia.org/wiki/Indentation_style#Variant:_1TBS_(OTBS)) I probably didn't follow through with it but would like to keep the brackets as they were.